freq/static/freq/js/freq.js

```diff
@@ -570,6 +571,18 @@ function logData(data, textStatus, jqXHR){
   console.log(data, textStatus, jqXHR);
 }
 
+var _gauges = _gauges || [];
```

Comment:
Kan je hier makkelijk iets aan toevoegen qua "test of er een variabele ergens is gezet" en dan die variabele in de django template stoppen? En die alleen aanzetten als debug mode uit staat? Want anders vervuilen de statistieken enorm met ons lokale getest.